<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerpetualData</class>
 <widget class="QMainWindow" name="PerpetualData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>364</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Perpetual Data</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>306</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuStatus">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <addaction name="actionAvailable"/>
     <addaction name="actionAway"/>
     <addaction name="actionBusy"/>
     <addaction name="actionOffline_2"/>
    </widget>
    <addaction name="actionLogout"/>
    <addaction name="actionQuit"/>
    <addaction name="menuStatus"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullScreen"/>
    <addaction name="actionMy_Files"/>
    <addaction name="actionPrivate_Shares"/>
    <addaction name="actionOffline"/>
    <addaction name="actionEmail"/>
   </widget>
   <widget class="QMenu" name="menuApplications">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionWord_Processor"/>
    <addaction name="actionSpread_Sheet"/>
    <addaction name="actionPresentation"/>
    <addaction name="actionBrowser"/>
    <addaction name="actionSettings_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuToolTip">
     <property name="title">
      <string>ToolTip</string>
     </property>
     <addaction name="actionOff_2"/>
     <addaction name="actionSmall_2"/>
     <addaction name="actionFull_2"/>
    </widget>
    <addaction name="actionManual"/>
    <addaction name="actionAbout"/>
    <addaction name="menuToolTip"/>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuApplications"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionWord_Processor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Word Processor</string>
   </property>
  </action>
  <action name="actionSpread_Sheet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Spread Sheet</string>
   </property>
  </action>
  <action name="actionPresentation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Presentation</string>
   </property>
  </action>
  <action name="actionBrowser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Browser</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionMy_Files">
   <property name="text">
    <string>My files</string>
   </property>
  </action>
  <action name="actionPrivate_Shares">
   <property name="text">
    <string>Private shares</string>
   </property>
  </action>
  <action name="actionOffline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Offline mode</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAvailable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Online</string>
   </property>
  </action>
  <action name="actionAway">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Away</string>
   </property>
  </action>
  <action name="actionBusy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Busy</string>
   </property>
  </action>
  <action name="actionOffline_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Offline</string>
   </property>
  </action>
  <action name="actionSettings_2">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionEmail">
   <property name="text">
    <string>Email</string>
   </property>
  </action>
  <action name="actionWhat_s_this">
   <property name="text">
    <string>What's this</string>
   </property>
   <property name="shortcut">
    <string>Shift+F11</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionSmall">
   <property name="text">
    <string>Small</string>
   </property>
  </action>
  <action name="actionFull">
   <property name="text">
    <string>Full</string>
   </property>
  </action>
  <action name="actionOff_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionSmall_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
  </action>
  <action name="actionFull_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Full</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
